org-formation/075-security-hub/security-hub-suppress-infra.yaml

```diff
@@ -379,6 +379,7 @@ Resources:
               - 'arn:aws:securityhub:::ruleset/cis-aws-foundations-benchmark/v/1.2.0/rule/4.1'
               - 'arn:aws:securityhub:::ruleset/cis-aws-foundations-benchmark/v/1.2.0/rule/4.2'
               - 'cis-aws-foundations-benchmark/v/1.4.0/3.5'  # (IT-3619) "3.5 Ensure AWS Config is enabled in all regions"
+              - 'cis-aws-foundations-benchmark/v/1.4.0/2.1.5.1'  # (IT-4193) "2.1.5.1 S3 Block Public Access setting should be enabled"
```

Comment:
I don't like the idea of suppressing this for all buckets in all accounts, how about adding more bucket prefixes to `SuppressFindingsForPublicBucketsRule`?

Reply:
Oh, I see, 2.1.5.1 is the account-level setting, and 2.1.5.2 is the bucket-level setting